config/prow/plugins.yaml

```diff
@@ -295,6 +297,17 @@ slack:
     - gardener-robot-ci-2
     - gardener-robot-ci-3
     - dependabot[bot]
+  - repos:
+    - gardener/gardener-landscape-kit
+    channels:
+    - prow-alerts
+    exempt_users:
+    - gardener-prow[bot]
+    - gardener-ci-robot
+    - gardener-robot-ci-1
+    - gardener-robot-ci-2
+    - gardener-robot-ci-3
+    - dependabot[bot]
 
```

Comment:
Something like [this](https://github.com/gardener/ci-infra/blob/d7cf6fe237adfb0772f09299d4fb7be12d8b302a/config/prow/plugins.yaml#L465-L469) is missing in `plugins.yaml` file.